extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -152,12 +154,21 @@ const renderOutlinkPageAttachmentUI = (
     openAttachmentEl.setAttribute('href', attachmentHref);
   }
 
-  // Setting theme
-  openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'));
-
   // Getting elements
   const {chipEl, ctaLabelEl} = htmlRefs(openAttachmentEl);
 
+  // Setting theme
+  let themeAttribute = attachmentEl.getAttribute('theme');
+  if (themeAttribute) {
+    themeAttribute = AttachmentTheme[themeAttribute.toUpperCase()];
+  }
```

Comment:
```suggestion
  let themeAttribute = attachmentEl.getAttribute('theme').toLowerCase();
```

Simplifying a bit, this should give you the same value

Reply:
This doesn't work bc theme isn't always set, so there needs to be a check. I went with:

```
let themeAttribute = attachmentEl.getAttribute('theme');
  if (themeAttribute) {
    themeAttribute = themeAttribute.toLowerCase();
  }
```